packages/flutter/lib/src/widgets/selection_container.dart

```diff
@@ -179,7 +179,21 @@ class _SelectionContainerState extends State<SelectionContainer> with Selectable
   @override
   SelectedContent? getSelectedContent() {
```

Comment:
This change is going to need a unit test for the updated behavior of `getSelectedContent()`

Reply:
Added in `selectable_text_test.dart`